CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ Write the date in place of the "Unreleased" in the case a new version is release
 # Changelog
 
 
+## Unreleased
+
+### Fixed
+
+- A regression in v0.1.1 disallowed specifying `trees` and `media_types`
```

Comment:
If the code is left to just `trees` and `specs`, this should be updated to match